Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -2078,6 +2078,31 @@ void TextureCacheBase::CopyRenderTargetToTexture(
   u32 tex_h = height;
   u32 scaled_tex_w = g_renderer->EFBToScaledX(width);
   u32 scaled_tex_h = g_renderer->EFBToScaledY(height);
+  bool EFBScaleExcludeTexture = false;
+
+  if (g_ActiveConfig.iEFBExcludeEnabled && !is_xfb_copy)
```

Comment:
On line 2115, there is another check if the copy should be upscaled or not which can be incorporated here.

Reply:
Thanks for the feedback, given that moving the check down to 2115 would allow scaleByHalf to override the base texture width and height, I just wanted to understand what that parameter actually means before pushing the change. Is scaleByHalf a part of standard Wii behaviour or a setting that EFBScaleExcludeTexture should force to ignore if true (unclear from a check for references)?

Reply:
It's part of standard Wii behavior, applied by games when they see the need to do so.  If the texture is not scaled (due to EFBScaleExcludeTexture), `scaleByHalf` should still be applied (in this case, it would go from 1x IR to .5IR).  If the texture _is_ scaled, then `scaleByHalf` still applies (reducing from e.g. 8x IR to 4x IR), but perhaps it shouldn't.

Reply:
Thanks, update pushed :)